src/style.css

```diff
@@ -0,0 +1,22 @@
+body {
+  margin: 0;
+}
+
+.red {
+  display: block;
+  height: 100px;
+  width: 100%;
```

Comment:
tag `div` by default has `width: 100%;` 
Don't write `width: 100%;` to block tags